dags/vds_pull_vdsdata.py

```diff
@@ -135,8 +129,12 @@
             op_kwargs = conns
         )
 
-        skip_update_inventories >> [pull_detector_inventory_task, pull_entity_locations_task]
+        t_done = ExternalTaskMarker(
+                task_id="done",
+                external_dag_id="vds_pull_vdsvehicledata",
+                external_task_id="starting_point"
+        )
+
+        [pull_detector_inventory_task, pull_entity_locations_task] >> t_done
 
-    check_partitions
-    vdsdata >> v15data #pull then summarize
-    update_inventories
+    update_inventories >> check_partitions >> vdsdata >> v15data #pull then summarize
```

Comment:
It might make more sense to have task group `update_inventories` defined at the beginning to reflect that it's the first task (group) in this DAG. But that doesn't affect anything in how the DAG works, i.e., you don't have to do what I'm suggesting